crates/hir_ty/src/display.rs

```diff
@@ -997,7 +998,19 @@ impl HirDisplay for TypeRef {
                 write!(f, "dyn ")?;
                 f.write_joined(bounds, " + ")?;
             }
-            TypeRef::Error | TypeRef::Macro(_) => write!(f, "{{error}}")?,
+            TypeRef::Macro(macro_call) => {
```

Comment:
This is quite unfortunate:

![image](https://user-images.githubusercontent.com/5961244/114410911-0675dd80-9bac-11eb-971e-e75c9c32d3c1.png)